core/tests/test_utils.py

```diff
@@ -71,11 +72,27 @@ def __init__(self, service_name='urlfetch'):
         self.request = None
 
     def set_return_values(self, content='', status_code=200, headers=None):
+        """Set mock content, status_code and headers for Url Service mock
```

Comment:
Also, here and elsewhere: please ensure that every docstring, arg definition and return value ends with a full stop.